instituto-brasileiro-de-informacao-em-ciencia-e-tecnologia-abnt.csl

```diff
@@ -165,121 +161,227 @@
       </substitute>
     </names>
   </macro>
-  <macro name="access">
+  <macro name="citation-locator">
+    <group>
+      <label variable="locator" form="short"/>
+      <text variable="locator" prefix=" "/>
+    </group>
+  </macro>
+  <macro name="collection-info">
     <choose>
-      <if variable="URL DOI" match="all">
-        <text variable="DOI" prefix="DOI " suffix=". "/>
-        <text term="available at" text-case="capitalize-first" prefix=" " suffix=": "/>
-        <text variable="URL"/>
+      <if type="book chapter entry-dictionary entry-encyclopedia map report" match="any">
+        <group>
+          <text value="("/>
```

Comment:
set these brackets as prefix+suffix on the group. that keeps them in one level and avoids "value"